src/ensembl/production/dbcopy/api/viewsets.py

```diff
@@ -27,6 +30,30 @@ class RequestJobViewSet(mixins.CreateModelMixin,
     pagination_class = None
     lookup_field = 'job_id'
 
+    def create(self, request, *args, **kwargs):
+        params = {
+            "src_host",
+            "src_incl_db",
+            "src_skip_db",
+            "src_incl_tables",
+            "src_skip_tables",
+            "tgt_host",
+            "tgt_db_name",
+        }
+        filters = {k: v for k, v in request.data.items() if k in params and v}
+        for job in RequestJob.equivalent_running_jobs(**filters):
```

Comment:
See below about this comment
```suggestion
        for job in RequestJob.objects.equivalent_running_jobs(**filters):
```

Reply:
Please see e529355